plugins/3-pytest.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+test -f pytest.ini || return 1
+
+pytest_test() {
+  run pytest
```

Comment:
If `Pipfile` is present, `$ pipenv run pytest` will use the locally installed version.